geniesp/bpc_redcap_export_mapping.py

```diff
@@ -707,15 +707,32 @@ def create_fixed_timeline_files(self, timeline_infodf, timeline_type):
             "used": used_entities,
         }
 
+    
+    def get_mg_synid(self, synid_folder: str, file_name: str) -> str:
+        """Get Synapse ID of main GENIE data file in release folder. 
+
+        Args:
+            synid_folder (str): Synapse ID of main GENIE release folder
+            file_name (str): File name for which to retrieve Synapse ID
+
+        Returns:
+            str: Synapse ID if file found; otherwise, None
+        """
+        synid_children = self.syn.getChildren(synid_folder)
+        for synid_child in synid_children:
+            if synid_child["name"] == file_name:
+                return synid_child["id"]
+        return None
```

Comment:
Did we want to put in a safe guard for if it is None?  The code will fail..

Reply:
yes...will implement

Reply:
implemented